iosApp/Launch Screen.storyboard

```diff
@@ -0,0 +1,89 @@
+            </objects>
+            <point key="canvasLocation" x="-280" y="-77"/>
+        </scene>
+        <!--Nearby-->
+        <scene sceneID="YPg-5f-9cT">
+            <objects>
+                <viewController id="sqa-ng-KwG" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="Kck-9R-Faz">
+                        <rect key="frame" x="0.0" y="0.0" width="393" height="769"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="empty-map-grid" translatesAutoresizingMaskIntoConstraints="NO" id="XRc-HZ-h9m">
+                                <rect key="frame" x="0.0" y="0.0" width="393" height="769"/>
+                            </imageView>
+                            <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="empty-search" translatesAutoresizingMaskIntoConstraints="NO" id="06y-uy-U7k">
+                                <rect key="frame" x="0.0" y="59" width="393" height="68"/>
+                            </imageView>
+                        </subviews>
+                        <viewLayoutGuide key="safeArea" id="HQS-pf-pis"/>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor"/>
+                        <constraints>
+                            <constraint firstItem="HQS-pf-pis" firstAttribute="trailing" secondItem="06y-uy-U7k" secondAttribute="trailing" id="10b-CO-xF9"/>
+                            <constraint firstItem="XRc-HZ-h9m" firstAttribute="leading" secondItem="HQS-pf-pis" secondAttribute="leading" id="4kV-Hg-oou"/>
+                            <constraint firstItem="06y-uy-U7k" firstAttribute="leading" secondItem="HQS-pf-pis" secondAttribute="leading" id="EmL-9b-JH7"/>
+                            <constraint firstItem="XRc-HZ-h9m" firstAttribute="top" secondItem="Kck-9R-Faz" secondAttribute="top" id="bR7-U6-QZV"/>
+                            <constraint firstItem="HQS-pf-pis" firstAttribute="trailing" secondItem="XRc-HZ-h9m" secondAttribute="trailing" id="m95-0D-EOv"/>
+                            <constraint firstAttribute="bottom" secondItem="XRc-HZ-h9m" secondAttribute="bottom" id="uKU-eB-wDX"/>
+                            <constraint firstItem="06y-uy-U7k" firstAttribute="top" secondItem="HQS-pf-pis" secondAttribute="top" id="yrr-2x-K91"/>
+                        </constraints>
+                    </view>
+                    <tabBarItem key="tabBarItem" title="Nearby" image="tab-icon-nearby" id="0Ae-j7-PL3"/>
```

Comment:
Sorry, missed this before it got merged, but we switched to using the `tab-icon-nearby` and `tab-icon-more` images for the tab bar, can we actually use images from our assets here? I think we also want to leave the text labels blank, since they could be localized to something different after the app loads.